tests/test_math.py

```diff
@@ -148,6 +148,8 @@ def test_log1mexp():
     with warnings.catch_warnings():
         warnings.filterwarnings("ignore", "divide by zero encountered in log", RuntimeWarning)
         warnings.filterwarnings("ignore", "invalid value encountered in log", RuntimeWarning)
+
+        warnings.warn("pymc.math.log1mexp_numpy is being deprecated.", FutureWarning)
```

Comment:
No need to issue the warning in the tests. Test code is not user facing. Same for all other warnings below.

Reply:
OK so I can take out those warnings and I presume then it is the same as the test_continuous case of just substituting in alternative code that achieves the same or is it that i should just undo my edits in test_math.py since they are test codes?

Reply:
This is different from the test_continuous. This is testing the functions directly, so can stay as is. This code will be removed when the respective functions are removed.

The `test_continuous` is the only one that needs a separate version of the function, that is implemented in the same file.